keras/layers/recurrent.py

```diff
@@ -2096,7 +2096,8 @@ class LSTM(RNN):
         return_sequences: Boolean. Whether to return the last output
             in the output sequence, or the full sequence.
         return_state: Boolean. Whether to return the last state
-            in addition to the output.
+            in addition to the output. The returned elements of the
```

Comment:
Thank you for the PR, @eyalzk. How about adding related descriptions into L293-299 (`class RNN`)? For example, `the number of state tensors is 1 (RNN and GRU) or 2 (LSTM)`.

Reply:
@eyalzk, Good point. I agree.

Reply:
Great! @taehoonlee 
Added a new commit.